.github/workflows/ci.yml

```diff
@@ -1,5 +1,11 @@
 name: CI
 on: [push, pull_request, workflow_dispatch]
+
+# Prevent intermediate images from being used by another run.
+concurrency:
+  group: ${{ github.ref }}
+  cancel-in-progress: true
```

Comment:
Hmm, I've tinkered with `concurrency` before, and two things stand out here:

1. I'm not sure we want to apply `cancel-in-progress` to runs on `master` (e.g. releases). Doing so will mean that two independent pushes to master (potentially separated by some time if workflow run times are long) which should ideally produce two different `build-*` tags will instead produce just the latest one. When "push to master" means "make a release", I think the relationship should stay 1:1. I think fine to cancel in progress jobs on branches and PRs though, since these are meant for testing/dev only.
2. Concurrency groups apply repo-wide, i.e. across workflows. It's a good idea to explicitly scope the group key to this workflow (e.g. `${{ github.workflow }}@${{ github.ref }}` or similar), since that's an assumed limitation here.

Reply:
I've never used this feature before, so thanks for those notes!

It should be fine to disable concurrency for just the default branch given independent pushes should produce unique `build-*` tags (near-zero chance of collision since the value is time-based). I'll look into that.

Reply:
[This force-push](https://github.com/nextstrain/docker-base/compare/68d46254213d48c8d93b8b2032527729e682ea0a..02b53264a0b08fa1e673afae4eb7432b55e56284) should address the above points. I did not test it, but added an entry in the PR description's post-merge tasks.